docs/features/software-catalog/api.md

```diff
@@ -504,6 +537,39 @@ Response type is JSON, on the form
 }
 ```
 
+### `GET /entity-facets?facet=<string>&facet=<string>&filter=<string>&filter=<string>`
+
+Get all entity facets that match the given filters.
+
+Response type is JSON, on the form
+
+```json
+{
+  "facets": {
+    "exercitation84": [
+      {
+        "value": "<string>",
+        "count": "<number>"
+      },
+      {
+        "value": "<string>",
+        "count": "<number>"
+      }
+    ],
+    "consectetur_1": [
+      {
+        "value": "<string>",
+        "count": "<number>"
+      },
+      {
+        "value": "<string>",
+        "count": "<number>"
+      }
+    ]
+  }
+}
```

Comment:
There's something off about this. Are you sure this is an example result?

Reply:
Corrected. This is referenced [here](https://github.com/backstage/backstage/blob/2c3f493ee32d67d3a300d13e73e352f91145a3ee/plugins/catalog-backend/src/schema/openapi.yaml#L402)

Reply:
Yup. Updated it.